test/editor/deserialize-test.ts

```diff
@@ -183,6 +183,14 @@ describe("editor/deserialize", function () {
             expect(parts[1]).toStrictEqual({ type: "user-pill", text: "Alice]", resourceId: "@alice:hs.tld" });
             expect(parts[2]).toStrictEqual({ type: "plain", text: "!" });
         });
+        it("user pill with displayname containing linebreak square bracket", function () {
+            const html = 'Hi <a href="https://matrix.to/#/@alice:hs.tld">Alice<br/>123</a>!';
```

Comment:
there is no square bracket in this input?

Reply:
Copy/paste error, apologies. 🙄